app/models/hit_count.rb

```diff
@@ -1,4 +1,4 @@
 class HitCount < ApplicationRecord
   belongs_to :user
-  scope :total_hit_count, ->{sum(:count)}
+  scope :total_hit_count, ->{ sum(:count) }
```

Comment:
Why this change
Nothing related to HitCount is mentioned in changes docs

Reply:
fixed